src/gui/DatabaseTabWidget.cpp

```diff
@@ -764,6 +767,11 @@ void DatabaseTabWidget::emitActivateDatabaseChanged()
     Q_EMIT activateDatabaseChanged(currentDatabaseWidget());
 }
 
+void DatabaseTabWidget::emitDatabaseUnlockedFromDbWidgetSender()
```

Comment:
But It's what it does, "emit databaseUnlocked from a widget sender".
Maybe only emitDatabaseUnlocked ?